packages/world/src/Utils.sol

```diff
@@ -13,14 +13,14 @@ function checkWorldStatus() view {
   require(!WorldStatus._getInMaintenance(), "Biomes is in maintenance mode. Try again later");
 }
 
-function inWorldBorder(VoxelCoord memory coord) pure returns (bool) {
+function inWorldBorder(Vec3 coord) pure returns (bool) {
   return
-    coord.x >= WORLD_BORDER_LOW_X &&
-    coord.x <= WORLD_BORDER_HIGH_X &&
-    coord.y >= WORLD_BORDER_LOW_Y &&
-    coord.y <= WORLD_BORDER_HIGH_Y &&
-    coord.z >= WORLD_BORDER_LOW_Z &&
-    coord.z <= WORLD_BORDER_HIGH_Z;
+    coord.x() >= WORLD_BORDER_LOW_X &&
```

Comment:
If we end up doing this in more places in the future could make this a util in coord, and just pass in WORLD_BORDER_LOW and WORLD_BORDER_HIGH cuz we can inline define them now I think cuz Vec3 is not a struct

Reply:
ah I wanted to talk about that, imo things related to other entities we probably want to keep outside vec3, as it starts becoming too coupled with everything else. What we can do for better syntax is to instead wrap the existing utils into libraries, and then use stuff like "using PlayerUtils for *" if necessary